wallet/src/de/schildbach/wallet/ui/buy_sell/BuyAndSellIntegrationsActivity.kt

```diff
@@ -210,9 +187,7 @@ class BuyAndSellIntegrationsActivity : LockScreenActivity() {
         viewModel.upholdBalanceLiveData.observe(this) {
             if (it != null) {
                 when (it.status) {
-                    Status.LOADING -> {
-                        loadingDialog?.show(this)
```

Comment:
We don't need to show the loading dialog when the balance is being fetched - just update UI when it's received.